packages/contracts-bridge/contracts/mocks/MockAccountant.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+
+import {NFTRecoveryAccountant} from "../accountants/NFTAccountant.sol";
+
+contract MockNftAccountant is NFTRecoveryAccountant {
+    mapping(address => bool) public allowList;
+
+    constructor() NFTRecoveryAccountant(address(0), address(1)) {}
+
+    function exposed_record(
```

Comment:
this will fail (`"overmint"`) unless we set `totalAffected` for some testnet tokens. 

it would be nice to set `totalAffected` anyway, as that's used in the GUI logic as well i think

Reply:
gonna add a public function to increase total affected

Reply:
I don't think I need totalAffected